crates/bevy_render/src/view/mod.rs

```diff
@@ -386,6 +389,19 @@ pub fn prepare_view_uniforms(
                 .unwrap_or_else(|| projection * inverse_view)
         };
 
+        let frustum = frustum
+            .map(|frustum| {
+                [
+                    frustum.half_spaces[0].normal_d(),
+                    frustum.half_spaces[1].normal_d(),
+                    frustum.half_spaces[2].normal_d(),
+                    frustum.half_spaces[3].normal_d(),
+                    frustum.half_spaces[4].normal_d(),
+                    frustum.half_spaces[5].normal_d(),
+                ]
+            })
```

Comment:
```suggestion
        .map(|frustum| frustum.half_spaces.map(|h| h.normal_d()))
```

Reply:
Maybe add a comment saying this is just converting it to a shader friendly format? Because it looks a bit out of place